player/lua/osc.lua

```diff
@@ -2072,6 +2086,17 @@ local function osc_init()
     bind_mouse_buttons("volume")
 
 
+    -- custom buttons
+    last_custom_button = 1
+    while user_opts["custom_button_" .. last_custom_button .. "_content"] and
+          user_opts["custom_button_" .. last_custom_button .. "_content"] ~= "" do
+        ne = new_element("custom_button_" .. last_custom_button, "button")
+        ne.content = user_opts["custom_button_" .. last_custom_button .. "_content"]
+        bind_mouse_buttons("custom_button_" .. last_custom_button)
+        last_custom_button = last_custom_button + 1
+    end
```

Comment:
not sure if that's important, but we can reduce number of lookups in user_opts. Something like, if you like it
```suggestion
    -- custom buttons
    for i = 1, math.huge do
        local content = user_opts["custom_button_" .. i .. "_content"]
        if not content or content == "" then
            break
        end
        ne = new_element("custom_button_" .. i, "button")
        ne.content = content
        bind_mouse_buttons("custom_button_" .. i)
        last_custom_button = i
    end
```